pipelines/VisualizeVisit.yaml

```diff
@@ -0,0 +1,14 @@
+description: Visualize full visit/focal plane with binned and mosaic images.
+tasks:
+  bin:
+    class: lsst.pipe.tasks.visualizeVisit.VisBinTask
+    config:
+      binning: 8
+      connections.inputExp: 'calexp'
+      connections.outputExp: 'calexpBin'
```

Comment:
Do you need to set these since it appears you are using the default names?

Reply:
Probably not.  It's likely an artifact from me changing those defaults before making the PR.